astropy/nddata/nddata.py

```diff
@@ -140,32 +139,32 @@ def __init__(self, data, uncertainty=None, mask=None, wcs=None,
             # other parameters.
             if (unit is not None and data.unit is not None and
                     unit != data.unit):
-                log.info("Overwriting NDData's current "
-                         "unit with specified unit")
+                log.info("overwriting NDData's current "
```

Comment:
I am not against this either, especially since there is a precedent. It is weird but it is not killing anyone, except maybe Grammar Police.